jupyterhub/tasks/main.yml

```diff
@@ -8,53 +8,42 @@
 
 
 - name: Install Python 3, Node, and NPM
-  apt: name={{item}} state=present update_cache=yes
+  apt: 
+     name: ['python3', 'python3-setuptools', 'python3-dev', 'python3-pip', 'sudo', 'npm', 'nodejs-legacy', 'git']
+     state: present 
+     update_cache: yes
   when: ansible_os_family == 'Debian'
-  with_items:
-  - python3
-  - python3-setuptools
-  - python3-dev
-  - python3-pip
-  - sudo
-  - npm
-  - nodejs-legacy
-  - git
   tags: jupyter-deps
 
 - name: Install Python 3, Node, and NPM
-  yum: name={{item}} state=present update_cache=yes
-  when: ansible_os_family == 'RedHat'
-  with_items:
-  - python36u
-  - python36u-setuptools
-  - python36u-devel
-  - python36u-pip
-  - sudo
-  - npm
-  - nodejs
-  - git
+  apt:
+     name: ['python37', 'python37-setuptools', 'python37-devel', 'python37-pip', 'sudo', 'npm', 'nodejs', 'git']
+     state: present 
+     update_cache: yes
+  when: ansible_os_family == 'Debian'
```

Comment:
Why is this Debian now instead of RedHat?

Reply:
I misunderstood the error output when I ran the playbook.  I've reversed the edit.

Reply:
I don't see the updated change in github. You may need to commit your changes (once you are happy with them) and push to github. Something like `git add jupyterhub/tasks/main.yaml`, then `git commit`, then `git push` - assuming you are still on this same branch.

Assuming in your code the command is `yum` and the family is "RedHat": Is `python37` a valid module name for CentOS 7?